### PR TITLE
Add two files on contributing

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Add two files on contributing:
- more details on how to contribute;
- Contributor info

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Contributors
+- Sijie Guo (https://github.com/sijie)
```
Comment (sijie) [inline Contributors.md]: use `[FirstName LastName](link)`

File: Contributors.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Jia Zhai (https://github.com/jiazhai)
+- Jennifer HUANG (https://github.com/Jennifer88huang)
```
Comment (sijie) [inline Contributors.md]:
```suggestion
- Jennifer Huang (https://github.com/Jennifer88huang)
```

File: Contributors.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- Bila ()
+- Yiming LI ()
```
Comment (sijie) [inline Contributors.md]:
```suggestion
- Yiming Li ()
```

File: Contributors.md (hunk 1 above)
Comment (sijie) [inline Contributors.md]: I would suggest sorting the list by first name in alphabet order

File: how-to-contribute.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# First Contributions
+It's always hard the first time you do something. Especially when you are collaborating, making mistakes isn't a comfortable thing. We wanted to simplify the way new open-source contributors learn and contribute for the first time.
+
+## How to contribute
+If you don't know how to use Git or how to make a pull request, refer to [First Contributions in Open Source project](https://github.com/firstcontributions/first-contributions).
+If you are not comfortable with tutorial, you can also watch a video on [Make a Pull Request](http://makeapullrequest.com/).
+
+## 第一次参与开源项目
```
Comment (sijie) [inline how-to-contribute.md]: should we separate Chinese from English here?

File: Contributors.md (hunk 1 above)
Comment (Jennifer88huang) [reply Contributors.md]: good idea.

File: how-to-contribute.md (hunk 4 above)
Comment (Jennifer88huang) [reply how-to-contribute.md]: good idea.

**Comment by sijie** [comment]
@Jennifer88huang 

>  adopt sijie's advice.

I didn't see the changes going through. have you pushed your changes?

File: Contributors.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Contributors
+- Sijie Guo (https://github.com/sijie)
+- Jia Zhai (https://github.com/jiazhai)
+- Jennifer Huang (https://github.com/Jennifer88huang)
+- Bila ()
```
Comment (jiazhai) [inline Contributors.md]: https://docs.google.com/spreadsheets/d/1Q_KRVQjozzS_9JjteC0RV5AZRSG7p5lxOX4K2JQ7TmQ/edit#gid=0
some of the user info is tracked in this doc.

**Comment by Jennifer88huang** [comment]
I've pushed my update, please check.

**Pull request merged**